notebooks/en/semantic_cache_chroma_vector_database.ipynb

```diff
@@ -0,0 +1,1389 @@
+{
```

Comment:
<ul><li>Typo: In&nbsp;metadatas, we can <strong>informa</strong> a list of topics.</li><li>"In the&nbsp;<strong>document</strong>&nbsp;we store the big text, it's a different column in each Dataset." This sentence does not make sense to me</li><li>Using the function <strong><em>add</em></strong> we need to inform, at least <strong><em>documents</em></strong>, <strong><em>metadatas</em></strong> and <strong><em>ids</em></strong>. =&gt; Correct the commas placement =&gt; "When using the function <strong><em>add</em></strong> we need to inform at least <strong><em>documents</em></strong>, <strong><em>metadatas,</em></strong> and <strong><em>ids"</em></strong></li></ul><p> </p><p> </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/huggingface/cookbook/pull/53/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>

Reply:
<p>Reformuled: </p><p> </p><p>We are now ready to add the data to the collection using the <code>add</code> function. This function requires three key pieces of information:</p><p> </p><p>* In the <strong>**document**</strong> we store the content of the <code>Answer</code> column in the Dataset. </p><p>* In <strong>**metadatas**</strong>, we can inform a list of topics. I used the value in the column <code>qtype</code>. </p><p>* In <strong>**id**</strong> we need to inform an unique identificator for each row. I'm creating the ID using the range of <code>MAX_ROWS</code>.</p>

Reply:
Thank you ! Very clear now!